cluster-autoscaler/core/scaledown/pdb/pdb.go

```diff
@@ -41,25 +39,31 @@ func NewPdbRemainingDisruptions(pdbs []*policyv1.PodDisruptionBudget) *PdbRemain
 	return &PdbRemainingDisruptions{pdbsCopy}
 }
 
-// CanDisrupt return if the pod can be removed.
-func (p *PdbRemainingDisruptions) CanDisrupt(pods []*apiv1.Pod) (bool, *drain.BlockingPod) {
+// CanDisrupt return if the set of pods can be removed.
+// isRisky means that the number of pods that belongs to one PDB, 
```

Comment:
nit: "isRisky means that not all pods can be removed in parallel."? Since this is exactly what it means. Maybe it should be called `inParallel` (with true/false meaning inverted) or something?

Reply:
Sure, but this is a problem of scale down implementation. `PdbRemainingDisruptions` just tells the caller whether the pods can be removed in parallel and it is up to the caller to decide whether or not to try and start node removal. The caller (i.e. scale down planner) is free to say `risky := !inParallel` in the code.

Reply:
I don't think we are going to use PdbRemainingDisruptions somewhere else, so I would like to use the same logic inside caller and implementation. Having risky := !inParallel add unnecessary complexity as for me.

Reply:
`PdbRemainingDisruptions` is generic enough to be useful beyond this class, but I guess we can always extract it later when actually needed.